packages/dapp/src/components/Boosts/Boost.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { State } from "../../redux/reducers";
+import { BoostHeaderWrapper, BoostHeader, BoostWrapper, ComingSoonText } from "./BoostStyledComponents";
+import { Helmet } from "react-helmet";
+import ScrollToTopOnMount from "../utility/ScrollToTop";
+import { Boost } from "@joincivil/civil-sdk";
+import { FeatureFlag } from "@joincivil/components";
+
+export interface BoostPageProps {
+  match: any;
+  boostId: string;
+}
+
+class BoostPage extends React.Component<BoostPageProps> {
+  public render(): JSX.Element {
+    return (
+      <>
+        <Helmet title="Civil Boost - The Civil Registry" />
+        <ScrollToTopOnMount />
+        <FeatureFlag feature={"boost"} replacement={<ComingSoonText />}>
```

Comment:
Do we want this coming soon text? Hadn't seen anything about it but I might have missed it. The default replacement is `<></>` which would also work.

Also, the feature flag nick mentioned in slack which I used in mine is `boosts-mvp`

Reply:
You can only reach this page if you know the url but I think it's better to say coming soon than have a completely blank page with no messaging

Reply:
Oh yeah good point, that makes sense!